test/integration/model.test.js

```diff
@@ -2172,7 +2172,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       const expectedLengths = {
         mssql: 2,
         postgres: 2,
-        db2: 10,
+        db2: 2,
```

Comment:
`dropTestSchemas` was broken so previous test schemas were not being purged. You'd get 8 extra schemas created by previous tests that were never deleted.

After this PR, only `[ 'special ', 'schema_test' ]` are returned because all other (non technical) schemas are properly deleted (`special` and `schema_test` are created by this test's `beforeEach`)